src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -565,18 +613,41 @@ void CropWindowTool::findCropWindowPlug()
 	SceneAlgo::History::Ptr history = SceneAlgo::history( scenePlug()->globalsPlug(), rootPath );
 	if( history )
 	{
-		if( !findCropWindowPlug( history.get(), /* enabledOnly = */ true ) )
+		const bool foundAnEnabledPlug = findCropWindowPlug( history.get(), /* enabledOnly = */ true );
+		// If we didn't find an enabled cropWindow plug upstream, or we did and it's
+		// read-only, look again for any other plugs that could be edited, but aren't
+		// enabled yet. We'll enable it if the user makes an edit.
+		if( !foundAnEnabledPlug || MetadataAlgo::readOnly( m_cropWindowPlug.get() ) )
 		{
 			findCropWindowPlug( history.get(), /* enabledOnly = */ false );
 		}
 	}
 
 	if( m_cropWindowPlug )
 	{
-		bool editable = m_cropWindowPlug->settable();
-		editable = m_cropWindowEnabledPlug ? editable || m_cropWindowEnabledPlug->settable() : editable;
-		m_overlay->setEditable( editable );
-		m_overlay->setCaption( m_cropWindowPlug->relativeName( m_cropWindowPlug->ancestor<ScriptNode>() ) );
+		const std::string plugName =  m_cropWindowPlug->relativeName( m_cropWindowPlug->ancestor<ScriptNode>() );
+
+		// If even after the second search, we could still be read-only
```

Comment:
"If even" -> "Even"

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3239/commits/558040aa1361bf1df48a471a037f3655bff30c97